18/README.md

```diff
@@ -2,211 +2,158 @@
+During the `end_block` call, all transactions about to be committed are checked. Every transfer of a vote token triggers a functions that counts the number of positive votes of Election over the number of voters. If the ratio is greater than ⅔, then the current Validator commits the change. Given the BFT nature of the system, all non-Byzantine Validator will commit the change at the same block height.
 
 Each Validator checks every new transaction that is about to be committed in a block. The process is roughly the following:
 1. If the transaction is **not** a valid Vote, return.
-4. If `asset.data.id` is **not** a valid Election, return.
-2. If the Election has less than ⅔ of positive votes, return.
-3. Execute the logic to implement the Election.
-
-###  Validate Vote
-A Validator must be able to discern valid Votes from invalid ones. The process is roughly the following:
-
-1. If the transaction is **not** a `TRANSFER`, return false.
-2. If `metadata.type` is **not** `vote`, return false.
-3. If `inputs.owners_before` is **not** a Validator, return false.
-5. Return true.
-
-### Validate Election
-A Validator must be able to discern valid Elections from invalid ones. The process is roughly the following:
-1. `type`: must contain the string `election`.
-2. `name`: name of the election.
-3. `version`: version number.
-4. `matter`: a human readable, short paragraph on the matter of change.
-5. If `inputs.owners_before` is **not** a Validator, return.
-6. If every `outputs.owners_after.amount` is **not** exactly `1`, return.
-7. If `outputs.owners_after` does **not** contain exactly all Validators, return.
-8. Apply any other validation rule enforced by the code specific to the election itself, return false if it fails.
-9. Return true.
+2. If `asset.data.id` is **not** a valid Election, return.
+3. If the Election has less than ⅔ of positive votes, return.
+4. If the Election has been already implemented, return.
+5. Since all previous checks has been successful, execute the logic to implement the Election.
+
+### What is a valid Vote?
+A Validator must be able to discern valid Votes from invalid ones. A valid vote is a transaction that spends one or more Vote Tokens to the Election Address.
+
+### What is a valid Election?
+A Validator must be able to discern valid Elections from invalid ones. A valid Election is a transaction where **all** the following conditions are true.
+
+1. `operation` is `CREATE`, or equivalent.
+2. Each public key listed in `inputs.owners_before` belongs to one of the Validators.
+3. `outputs.amount` is the total power of the Validators.
+4. `outputs` has as many entries as the total number of Validators.
+5. Each Validator is represented in `outputs`.
+6. Each entry in `outputs` can be spent by only one Validator, and the amount attached to it is equal to the power of that Validator.
+
+**Note: any change in the Validator Set will make old Elections invalid.**
```

Comment:
It seems to belong to the `What is a valid Vote?` section. The vote is valid if it transfers some vote tokens to a valid election and the validator set has not changed since the election was placed. Does it sound good?